common/character_interactions/00_prison_interactions.txt

```diff
@@ -3546,7 +3546,26 @@ execute_prisoner_interaction = {
 				NOT = { has_character_flag = is_being_tortured }
 			}
 		}
-	}
+		# Warcraft
+		# Elunic Kaldorei AI will not execute Kaldorei Illidari prisoners # Hotfix until new criminal doctrines
+		trigger_if = {
+			limit = { 
+				scope:actor = { is_ai = yes }
+				scope:actor.faith = faith:kaldorei_religion 
+				scope:actor = { has_trait = creature_night_elf }
+				scope:actor.culture = scope:recipient.culture 
+				scope:recipient.faith = faith:illidari 
+			}
+			NOR = {
+				scope:recipient = { 
+					has_trait = being_demon	
+				}
+				scope:recipient = { 
+					has_trait = creature_night_elf 
+				}
+			}
```

Comment:
Don't understand why you need so many checks. This will be enough.
```suggestion
				scope:actor.faith = faith:kaldorei_religion
			}
			scope:recipient.faith = faith:illidari 
```

Reply:
These were meant to ensure the interactions would be specific to recipients of Kaldorei Illidari of Darnassisn and by actors of Elunic Kaldorei of Darnassian culture. But as discussed, we can include that change in new crime doctrines.